AnaMunoz-Fatima-Functions-0826.js

```diff
@@ -0,0 +1,132 @@
+//============================================
+// PART 2 
+//============================================
+
+
+// # 1 Calculate the Cube
+//============================================
+const calculateCube = (x) => {
+  return x**3;
+}
+console.log(calculateCube(5));
+
+
+// #2 Is a Vowel?
+//============================================
+const isAVowel = (letter) => {
+  if (letter === 'a') {
+  return true
+} 
+else {
+return false
+}
+}
+console.log(isAVowel("a"));
+// console.log(isAVowel("b"));//Test
+
+// // #3 Get Two Lenghts?
+//============================================
+
+const getTwoLengths = (word1,word2) => {
+  return [word1,word2].map((x) => x.length); //The map() method calls the provided function once for each element in an array, in order
+} 
+console.log(getTwoLengths("Hank", "Hippopopalous"));
```

Comment:
nice use of `map()`! however, we hadn't talked about `map` when this assignment was given. this problem is simple enough that it shouldn't be too much trouble to re-do it without the array method.